include/libtorrent/add_torrent_params.hpp

```diff
@@ -151,7 +151,7 @@ namespace libtorrent
 			// in this ``add_torrent_params`` object will take precedence over
 			// whatever states are saved in the resume data. For instance, the
 			// ``paused``, ``auto_managed``, ``sequential_download``, ``seed_mode``,
-			// ``super_seeding``, ``max_uploads``, ``max_connections``,
+			// ``super_seeding``, ``priority``, ``max_uploads``, ``max_connections``,
```

Comment:
the semantics of the override resume data flag is that attributes/properties set for the torrent in this struct (add_torrent_params) will override any value loaded from the resume data. Since you cannot set the torrent's priority in add_torrent_params, there's nothing to override with.